miso-web/src/main/webapp/scripts/sample_hot.js

```diff
@@ -1084,38 +1050,49 @@ Sample.hot = {
     if (obj.groupDescription && obj.groupDescription.length) {
       sample.groupDescription = obj.groupDescription;
     }
-    
+
     // add SampleCategory-specific attributes.
     switch (Sample.hot.getCategoryFromClassId(sample.sampleClassId)) {
-    case 'Analyte':
+    case 'Aliquot':
       if (obj.samplePurposeAlias && obj.samplePurposeAlias.length) {
         sample.samplePurposeId = Hot.getIdFromAlias(obj.samplePurposeAlias, Hot.sampleOptions.samplePurposesDtos);
       }
       if (obj.sampleGroupComposite && obj.sampleGroupComposite.length) {
         sample.sampleGroupId = Sample.hot.getIdFromSGComposite(obj.sampleGroupComposite, Hot.sampleOptions.sampleGroupsDtos);
       }
-      if (obj.tissueMaterialAlias && obj.tissueMaterialAlias.length) {
-        sample.tissueMaterialId = Hot.getIdFromAlias(obj.tissueMaterialAlias, Hot.sampleOptions.tissueMaterialsDtos);
-      }
+      break;
+    case 'Stock':
       if (obj.strStatus && obj.strStatus.length) {
         sample.strStatus = obj.strStatus;
       }
-      if (obj.region && obj.region.length) {
-        sample.region = obj.region;
+      break;
+    case 'Tissue':
```

Comment:
Tissue fields should all still be included even if not creating a Tissue because they're needed for creating missing parents too